recipes/retworkx/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "retworkx" %}
+{% set version = "0.10.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/retworkx-{{ version }}.tar.gz
+    sha256: ba81cb527de7ff338575905bb6fcbebdf2ab18ae800169a77ab863f855bf0951
+
+build:
+  number: 0
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+   - {{ compiler('cxx') }}
+   - {{ compiler('rust') }}
+   - cargo-bundle-licenses
+   - make
+  host:
+    - pip
+    - python
+    - setuptools-rust
+  run:
+    - numpy >=1.16.0
+    - python
+
+test:
+  imports:
+    - retworkx
+    - retworkx.visualization
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Qiskit/retworkx
+  summary: A python graph library implemented in Rust
+  license: Apache-2.0
+  license_file:
+    - LICENSE
```

Comment:
Maybe add a note that next version will have the license:

https://github.com/Qiskit/retworkx/pull/500

(good show!)

Reply:
Good idea! Will do that if it doesn't get merged now or as an issue later.

Reply:
@bollwyvl Anything else I should change? I know you're not a reviewer but you're in every rust issue and PR I looked at. 😄

Reply:
:rofl: it's all so easy now! i love it.

Reply:
Yeah `cargo-bundle-licenses` is a gamechanger!

Reply:
i can't resolve, but the license stuff LGTM now...

Reply:
Cool thanks. You wouldn't know by the way who the person "responsible" for rust is, would you? OR the one normally reviewing them? Because there is no team for it yet and no one in Gitter.

Reply:
It's volunteers all the way down.... :turtle:

Reply:
So, the licenses end up in the package when using cargo-bundle-licenses? Cool.

Reply:
Yeah it's super neat! And the maintainer is active as well so looks like a great solution!